### ISSUE TITLE
CommonJS plugins are deprecated

### ISSUE THREAD

**Issue by Luquatic** [csstools/stylelint-use-logical#27]
when running `stylelint` I'm getting the following warning:

```
CommonJS plugins are deprecated ("/Users/.../node_modules/.pnpm/stylelint-use-logical@2.1.2_stylelint@16.2.1/node_modules/stylelint-use-logical/index.cjs"). See https://stylelint.io/migration-guide/to-16
```

**Installed Packages**
```
    "stylelint": "^16.2.1",
    "stylelint-config-prettier-scss": "^1.0.0",
    "stylelint-config-recommended-vue": "^1.5.0",
    "stylelint-config-standard-scss": "^13.0.0",
    "stylelint-use-logical": "^2.1.2",
```

**Comment by dflynn15** [comment]
Unfortunately, this is an issue with Stylelint, not this plugin. The fix PR in Stylelint is here: https://github.com/stylelint/stylelint/pull/7532. However, it still has not been released :(

**Comment by Luquatic** [comment]
Thanks, seems to have been released now with [16.3.0](https://github.com/stylelint/stylelint/releases/tag/16.3.0)!

**Comment by Luquatic** [comment]
Thanks, seems to have been released now with [16.3.0](https://github.com/stylelint/stylelint/releases/tag/16.3.0)!
